examples/js/loaders/GLTF2Loader.js

```diff
@@ -385,6 +392,24 @@ THREE.GLTF2Loader = ( function () {
 
 	}
 
+	/* DRACO MESH COMPRESSION EXTENSION */
+
+	function GLTFDracoMeshCompressionExtension () {
+
+		this.name = EXTENSIONS.KHR_DRACO_MESH_COMPRESSION;
+
+		this.dracoLoader = new THREE.DRACOLoader();
+
+	}
+
+	GLTFDracoMeshCompressionExtension.prototype.decodePrimitive = function ( primitive, dependencies ) {
+
+		var bufferViewID = primitive.extensions[ this.name ].bufferView;
```

Comment:
one problem is that our draco_decoder.js  currently does not provide API to query its version. I'll try to add that today.

Another thing is that we should most likely not care about the revision version, only the major and minor versions should affect the bitstream .. i.e. we should probably mention in the spec that for conformance, the decoder needs to support the major.minor version of the encoded file, revision can be ignored (if we want to include it at all).